sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -79,13 +79,65 @@ enum {
 
 #define SOF_SDW_MAX_DAI_NUM		8
 
+static const struct snd_soc_dapm_widget cs_spk_widgets[] = {
+	SND_SOC_DAPM_SPK("Speakers", NULL),
+};
```

Comment:
I don't really see any problem with updating the cirrus parts to use the generic speaker widget. The name of the widget does not currently form any sort of ABI and we will gain a PIN_SWITCH we probably should have anyway. Minor update would be needed in sof_sdw_cs_amp, but that is it. Unless @rfvirgil has any objections?

Reply:
I agree.  "Speakers" -> "Speaker" doesn't matter, there isn't anything depending on the name.
So sof_sdw_cs_amp can use generic_spk_widgets.

Reply:
> I agree. "Speakers" -> "Speaker" doesn't matter, there isn't anything depending on the name. So sof_sdw_cs_amp can use generic_spk_widgets.

Thanks for the comment. Let me change it to generic_spk_widgets.